source/features/select-notifications.css

```diff
@@ -1,5 +1,9 @@
-.js-notifications-mark-selected-actions > *,
-.rgh-open-selected-button {
-	/* Allow clicking on "Done" and other toolbar buttons even when the dropdown (z-index of 99) is open #4753 */
-	z-index: 100;
+/* Only when the meny is shown as a dropdown, not as a modal (mobile) #5796 */
```

Comment:
Meny?

Reply:
https://meny.no 😂

Reply:
When you think _it's just a one-line change_ but it has 5 bugs.